scripts/detect_errors.py

```diff
@@ -10,14 +10,14 @@ def encode_msg_text_for_github(msg):
 
 def format_msg(msg):
     # Formatted for https://github.com/actions/toolkit/blob/master/docs/commands.md#log-level
-    
+
     # mapping between lean severity levels and github levels.
     # github does not support info levels, which are emitted by `#check` etc:
     # https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-a-debug-message
     severity_map = {'information': 'warning'}
     severity = msg.get('severity')
     severity = severity_map.get(severity, severity)
-    
+
```

Comment:
This feels like scope creep!

Reply:
I have no idea how that happened! Never opened that file.